config-example/cl/config.yaml

```diff
@@ -132,7 +132,23 @@ MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # `6`
 BLOB_SIDECAR_SUBNET_COUNT: 6
 ## `uint64(6)`
-MAX_BLOBS_PER_BLOCK: $MAX_BLOBS_PER_BLOCK
+MAX_BLOBS_PER_BLOCK: 6
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
+MAX_REQUEST_BLOB_SIDECARS: 768
+
+# Electra
+# 2**7 * 10**9 (= 128,000,000,000)
+MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: 128000000000
+# 2**8 * 10**9 (= 256,000,000,000)
+MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT: 256000000000
+# `9`
+BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: $MAX_BLOBS_PER_BLOCK_ELECTRA
+# `uint64(6)`
+TARGET_BLOBS_PER_BLOCK_ELECTRA: $TARGET_BLOBS_PER_BLOCK_ELECTRA
+# `uint64(9)`
+MAX_BLOBS_PER_BLOCK_ELECTRA: $MAX_BLOBS_PER_BLOCK_ELECTRA
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA
+MAX_REQUEST_BLOB_SIDECARS_ELECTRA: $(($MAX_REQUEST_BLOCKS_DENEB * $MAX_BLOBS_PER_BLOCK_ELECTRA))
```

Comment:
that syntax won't work as we're using that weird envsubst.py hack for this file too?
https://github.com/ethpandaops/ethereum-genesis-generator/blob/master/entrypoint.sh#L81

Reply:
it is indeed not working :D maybe some bash magic with your solution will work?

Reply:
Should be fixed now.